.github/workflows/check.yml

```diff
@@ -45,6 +45,9 @@ jobs:
           make install
           poetry lock --no-update
           [ -z "$(git status --porcelain=v1 2>/dev/null)" ] || (echo "Lock file is not up to date, please run 'poetry lock --no-update'" && exit 1)
+      - name: Help on job failure
+        if: ${{ failure() }}
+        run: echo "Try running \"poetry cache clear --all pypi\" before \"poetry lock --no-update\""
```

Comment:
I don't think this will work - it will just be a collapsed step so no one will notice. Probably better to fold this into the `echo` in the previous step

Reply:
yeah... no one will notice this
<img width="1129" alt="Screenshot 2022-07-01 at 12 40 55" src="https://user-images.githubusercontent.com/86921811/176879443-96f0448b-02d0-4b37-b977-1d8b778c8924.png">